VERSION

```diff
@@ -1 +1 @@
-0.2.16
+0.2.17dev
```

Comment:
You should not add any non-number to SemVers.

If you want to set the version for dev, you need to set the minor version to odd.

For example, set it to `0.3.1`, then it will be the dev version when publishing to PyPI with our GitHub Actions DevOps script.

Reply:
I'am so careless that I ignore the SemVers rules. I have made it passed now.